tb_plugin/torch_tb_profiler/profiler/run_generator.py

```diff
@@ -367,7 +377,7 @@ def _generate_gpu_info(self):
 
                 mem = device_prop.get("totalGlobalMem")
                 if mem is not None:
-                    gpu_info["Memory"] = mem
+                    gpu_info["Memory"] = str(int(mem / 1000000000)) + 'G'
```

Comment:
"{} GB".format(round(float(mem) / 1024 / 1024 / 1024, 2))
I remember that some GPU card's memory are not exactly integer.